src/Test.sol

```diff
@@ -580,3 +580,49 @@ library stdStorage {
         return result;
     }
 }
+
+/*//////////////////////////////////////////////////////////////////////////
+                                DELTA MATHS
+//////////////////////////////////////////////////////////////////////////*/
+
+library deltaMaths {
+    function abs(int256 a) internal pure returns (uint256) {
+        if (a < 0) {
+            // lifted from: @OpenZeppelin/utils/math/SignedMath.sol
+            unchecked {
+                return uint256(-a);
+            }
+        }
+
+        return uint256(a);
+    }
+
+    function getDelta(uint256 a, uint256 b) internal pure returns (uint256) {
```

Comment:
Consistent with conventions like naming functions `max` instead of `getMax`
```suggestion
    function delta(uint256 a, uint256 b) internal pure returns (uint256) {
```

Reply:
Done, can resolve if happy